src/OSPSuite.Presentation/Presenters/SensitivityAnalyses/SensitivityAnalysisParameterSelectionPresenter.cs

```diff
@@ -107,12 +117,20 @@ public void ChangeSimulation(ISimulation simulation)
 
       public bool ValidateSimulationChange(ISimulation newSimulation)
       {
-         return _sensitivityAnalysisTask.ValidateSwap(_sensitivityAnalysis, _sensitivityAnalysis.Simulation, newSimulation);
+         var swapValidationResult = _sensitivityAnalysisTask.ValidateSwap(_sensitivityAnalysis, _sensitivityAnalysis.Simulation, newSimulation);
+
+         if (swapValidationResult.ValidationState == ValidationState.Valid)
+            return true;
+
+         using (var validationMessagesPresenter = _applicationController.Start<IValidationMessagesPresenter>())
```

Comment:
move the logic that was in the task in the presenter. That way I can resue the task for R